cellpack/autopack/Analysis.py

```diff
@@ -1295,6 +1311,18 @@ def run_analysis_workflow(
                 **analysis_config["create_report"],
             )
 
+    def normalize_similarity_df(self, similarity_df):
+        """
+        Normalizes the similarity dataframe
+        """
+        dims_to_normalize = self.get_list_of_dims() + ["pairwise_distance"]
+        for dim in dims_to_normalize:
+            values = similarity_df.loc[:, dim].values
+            similarity_df.loc[:, dim] = (values - numpy.min(values)) / (
+                numpy.max(values) - numpy.min(values)
```

Comment:
will there be case like: numpy.max(values) - numpy.min(values) == 0? should we check to avoid division by zero errors?

Reply:
Good point! I can add a check for this